cmd/netgoal/generate.go

```diff
@@ -61,6 +67,12 @@ func init() {
 	generateCmd.Flags().StringVarP(&nodeTemplatePath, "node-template", "", "", "json for one node")
 	generateCmd.Flags().StringVarP(&nonParticipatingNodeTemplatePath, "non-participating-node-template", "", "", "json for non participating node")
 	generateCmd.Flags().StringVarP(&relayTemplatePath, "relay-template", "", "", "json for a relay node")
+	generateCmd.Flags().StringVarP(&sourceWallet, "wallet-name", "", "", "Source wallet name")
+	generateCmd.Flags().IntVarP(&rounds, "rounds", "", 5000, "Number of rounds")
+	generateCmd.Flags().IntVarP(&roundTrxCount, "ntrx", "", 1000, "Transaction count")
+	generateCmd.Flags().IntVarP(&accountsCount, "naccounts", "", 100000, "Account count")
+	generateCmd.Flags().IntVarP(&assetsCount, "nassets", "", 100000, "Asset count")
+	generateCmd.Flags().IntVarP(&applicationCount, "napps", "", 100000, "Application Count")
```

Comment:
I'm pretty sure that we don't want to use these numbers as the default values. 
The rounds would clearly need to be zero ( so it would be backward compatible ).